src/processors/transform.rs

```diff
@@ -0,0 +1,430 @@
+
+extern crate cgmath;
+// extern crate test;
+
+use self::cgmath::{Quaternion, Vector3, Matrix3, Matrix4};
+
+use ecs::{Join, Component, NullStorage, VecStorage, Entity, RunArg, Processor};
+use context::Context;
+use std::sync::{Mutex, Arc};
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::collections::{HashMap, HashSet};
+
+/// Local position and rotation from parent.
+#[derive(Debug)]
+pub struct LocalTransform {
+    pos: [f32; 3], // translation vector
+    rot: [f32; 4], // quaternion [w (scalar), x, y, z]
+    scale: [f32; 3], // scale vector
+    dirty: AtomicBool,
```

Comment:
Yeah, Cell and Refell don't implement sync which Component requires and using an atomic let's us not lock the storage mutably.